src/contextify.cc

```diff
@@ -38,83 +38,85 @@ class ContextifyContext : public ObjectWrap {
-        ljsTmpl->SetClassName(NanNew("ContextifyContext"));
-        NODE_SET_PROTOTYPE_METHOD(ljsTmpl, "run",       ContextifyContext::Run);
-        NODE_SET_PROTOTYPE_METHOD(ljsTmpl, "getGlobal", ContextifyContext::GetGlobal);
+        ljsTmpl->SetClassName(className);
+        Nan::SetPrototypeMethod(ljsTmpl, "run",       ContextifyContext::Run);
+        Nan::SetPrototypeMethod(ljsTmpl, "getGlobal", ContextifyContext::GetGlobal);
 
-        NanAssignPersistent(jsTmpl, ljsTmpl);
-        target->Set(NanNew("ContextifyContext"), ljsTmpl->GetFunction());
+        jsTmpl.Reset(ljsTmpl);
+        target->Set(className, ljsTmpl->GetFunction());
     }
 
     static NAN_METHOD(New) {
-        NanScope();
-
-        if (args.Length() < 1) {
-            NanThrowError("Wrong number of arguments passed to ContextifyContext constructor");
-            NanReturnUndefined();
+        if (info.Length() < 1) {
+            Nan::ThrowError("Wrong number of arguments passed to ContextifyContext constructor");
+            info.GetReturnValue().SetUndefined();
+            return;
         }
 
-        if (!args[0]->IsObject()) {
-            NanThrowTypeError("Argument to ContextifyContext constructor must be an object.");
-            NanReturnUndefined();
+        if (!info[0]->IsObject()) {
+            Nan::ThrowTypeError("Argument to ContextifyContext constructor must be an object.");
+            info.GetReturnValue().SetUndefined();
+            return;
         }
 
-        ContextifyContext* ctx = new ContextifyContext(args[0]->ToObject());
-        ctx->Wrap(args.This());
-        NanReturnValue(args.This());
+        ContextifyContext* ctx = new ContextifyContext(info[0]->ToObject());
+        ctx->Wrap(info.This());
+        info.GetReturnValue().Set(info.This());
     }
 
     static NAN_METHOD(Run) {
-        NanScope();
-        if (args.Length() == 0) {
-            NanThrowError("Must supply at least 1 argument to run");
+        if (info.Length() == 0) {
+            Nan::ThrowError("Must supply at least 1 argument to run");
+            info.GetReturnValue().SetUndefined();
+            return;
         }
-        if (!args[0]->IsString()) {
-            NanThrowTypeError("First argument to run must be a String.");
-            NanReturnUndefined();
+        if (!info[0]->IsString()) {
+            Nan::ThrowTypeError("First argument to run must be a String.");
+            info.GetReturnValue().SetUndefined();
+            return;
         }
-        ContextifyContext* ctx = ObjectWrap::Unwrap<ContextifyContext>(args.This());
-        Local<Context> lcontext = NanNew(ctx->context);
-        lcontext->Enter();
-        Local<String> code = args[0]->ToString();
+        ContextifyContext* ctx = ObjectWrap::Unwrap<ContextifyContext>(info.This());
+        Local<Context> lcontext = Nan::New(ctx->context);
+        Context::Scope ctxScope(lcontext);
+        Local<String> code = info[0]->ToString();
 
         TryCatch trycatch;
```

Comment:
`Nan::TryCatch` exists since the constructor will require an `Isolate` soon, but the old constructor has not been officially deprecated yet, so it should be safe until Node 6 comes in April.